ComPact/Verifier/PactVerifier.cs

```diff
@@ -53,13 +53,32 @@ public async Task VerifyPactAsync(string path)
 
             if (pact.Interactions != null)
             {
-                if (_config.ProviderBaseUrl == null)
-                {
-                    throw new PactException("Could not verify pacts. Please configure a ProviderBaseUrl.");
+                if (_config.ProviderBaseUrl == null && _config.ProviderHttpClient == null)
```

Comment:
What would happen if a ProviderHttpClient is provided but not a ProviderBaseUrl? There is currently no check for that.

Reply:
That's exactly how I'm using it :) I pass in an ProviderHttpClient which is pre-configured and I don't pass in a ProviderBaseUrl (because the ProviderHttpClient is pre-configured by AspNet TestServer which gives you an HttpClient to use).

I was considering trying to add a check here to see if ProviderHttpClient is configured, but it started getting pretty gnarly, so in the end I figured that if someone passes in a client, it's probably reasonable to assume they'll have configured it? If they haven't, they'll soon realise they need to when none of their pacts validate :)